mne/utils/docs.py

```diff
@@ -167,17 +167,15 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 .. versionadded:: 0.24
 """
 
-# Cropping
+# tmin / tmax
 docdict['include_tmax'] = """
 include_tmax : bool
-    If True (default), include tmax. If False, exclude tmax (similar to how
-    Python indexing typically works).
-
-    .. versionadded:: 0.19
+    If True (default), include ``tmax``. If False, exclude ``tmax`` (similar to
+    how Python indexing typically works).
 """
 docdict['notes_tmax_included_by_default'] = """
 Unlike Python slices, MNE time intervals by default include **both**
-their end points; ``crop(tmin, tmax)`` returns the interval
+their end points; ``inst.method(tmin, tmax)`` returns the interval
```

Comment:
I find it less explicit for new comers what this inst.method means.

my 2c

Reply:
True, I'll make a separate note for crop and get_data